components/Icon.tsx

```diff
@@ -0,0 +1,50 @@
+type Props = {
+  type: string;
+};
+
+export default function Icon({ type }: Props) {
+  function setType() {
+    if (type === "home") {
+      return (
+        <>
+          <svg
+            className="h-7 w-7"
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            stroke="currentColor"
+            strokeWidth={1.5}
+            viewBox="0 0 24 24"
+            type="home"
```

Comment:
I don't think this prop exists on svgs 🤔 I'd remove it either way, doesn't do anything

Reply:
I was using it to define it. Dumb mistake. I didn't need to cuz we're sending the prop of type to Icon not to the svg